style.css

```diff
@@ -0,0 +1,417 @@
+/*
+* Prefixed by https://autoprefixer.github.io
+* PostCSS: v8.3.6,
+* Autoprefixer: v10.3.1
+* Browsers: last 4 version
+*/
+
+* {
+	-webkit-box-sizing: border-box;
+	        box-sizing: border-box;
+}
+
+html {
+	font-size: 62.5%;
+	font-family: 'Work Sans',sans-serif;
+}
+
+a.anchor {
+	display: block;
+	position: relative;
+	top: -70px;
+	visibility: hidden;
+}
+
+.lang {
+	font-family: monospace;
+}
+
+
+/* -- Nav --*/
+
+nav {
+	position: fixed;
+	top: 0;
+	left: 0;
+	display: -webkit-box;
+	display: -ms-flexbox;
+	display: flex;
+	-webkit-box-pack: justify;
+	    -ms-flex-pack: justify;
+	        justify-content: space-between;
+	-webkit-box-align: center;
+	    -ms-flex-align: center;
+	        align-items: center;
+	width: 100vw;
+	padding: 35px 10%;
+	background-color: #ffffff;
+	height: 40px;
+	-webkit-box-shadow:  0px 12px 24px -12px rgba(0, 0, 0, 0.5);
+	box-shadow:  0px 12px 24px -12px rgba(0, 0, 0, 0.5);
+}
+
+.navLogo {
+	color: #000000;
+	text-transform: uppercase;
+	text-decoration: none;
+	font-size: 1.9rem;
+	font-weight: 500;
+}
+
+.navLink {
+	color: #000000;
+	font-size: 1.6rem;
+	font-weight: bold;
+	text-transform: uppercase;
+	text-decoration: none;
+	margin-left: 10px;
+}
+
+.navLink:hover {
+	border-bottom: 4px solid #000000;
+}
+
+.burgerMenu {
+	display:none;
+}
+
+.burgerMenu div {
+	width: 30px;
+	height: 3px;
+	background-color: #000000;
+	margin: 6px 0;
+}
+
+@media only screen and (max-width: 664px) {
+	.navLink {
+		display: none;
+	}
+	
+	.burgerMenu {
+		display: block;
+		padding: 5px;
+	}
+}
+
+/* -- Header/Intro -- */
+
+header {
+	font-family: 'Work Sans', sans-serif;
+	margin-top: 70px;
+	width: 100vw;
+	height: calc(80vh - 70px);
```

Comment:
this is a clever way to achieve this. I would hesitate in future to use too much programmy snazz in css as it makes it a little unaccessible to anyone without good CSS skills.